### DISCUSSION TITLE
作者简介里研究成果格式遇到问题

### DISCUSSION THREAD

**Discussion by Estrellas3970** [note286/xduts#172] [Q&A] [answered]
下图是学校的word版格式
![Snipaste_2024-03-12_10-02-42](https://github.com/note286/xduts/assets/163080538/79ff8140-05e8-49cd-985e-041179eb1c20)
下图xduts的格式，没有悬挂缩进，并且没有自动换行，求教如何解决，万分感谢！
![Snipaste_2024-03-12_10-19-25](https://github.com/note286/xduts/assets/163080538/d195c78e-efec-4ecf-aad7-e2e2f5a69ab8)

**Comment by Estrellas3970** [comment]
遇到连续的数字或英文时，不会自动换行

**Reply by note286** [reply, marked as answer]
请阅读xduts手册关于作者简介配置的介绍。

请阅读`seqsplit`手册，使用`\seqsplit`命令。

**Reply by Estrellas3970** [reply]
感谢！！
